src/simulation/m_cbc.fpp

```diff
@@ -1303,7 +1302,7 @@ contains
 
             if(riemann_solver == 1) then
 !$acc parallel loop collapse(4) gang vector default(present)
-                do i = advxb, advxe
+                do i = 1, advxe
                     do r = is3%beg, is3%end
                         do k = is2%beg, is2%end
```

Comment:
where does this come from?

Reply:
I believe these are incorrect as well, I was certain these had 1, sys_size at first which was giving a bug, which I then corrected to 1, advxe. But it really should be advxb, advxe (it was perhaps corrected at some point, but the merge with my branch messed it up)

I'll fix these in the new commit